lib/pci_caps.c

```diff
@@ -292,6 +292,44 @@ handle_px_pxdc_write(vfu_ctx_t *vfu_ctx, struct pxcap *px,
     return 0;
 }
 
+static int
+handle_px_pxlc_write(vfu_ctx_t *vfu_ctx UNUSED, struct pxcap *px,
+                     const union pxlc *const p)
+{
+    assert(px != NULL);
+    assert(p != NULL);
+
+    /* FIXME RCB only writeable for endpoints and bridges */
+    /* FIXME must preserve reserved bits */
+    px->pxlc.raw = p->raw;
```

Comment:
I think it's probably better to just ignore the write?
That way at least e.g. .rl behaviour is preserved.

Reply:
sure